woocommerce/woocommerce-android
# Free Trial: Connect summary to store creation

Summary
==========
Fix issue #8757 by connecting the Free Trial creation summary to all possible store creation flow.

Capture
==========
https://user-images.githubusercontent.com/5920403/230453654-c938e07c-7362-4c75-a0dc-96d6e8410fec.mp4


How to Test
==========
1. Try to Create a Free Trial store and make sure there's no way to create it without interacting with the Summary view.

Update release notes:
